front-end-poc/src/ModelLookup/pmml/LinearRegressionViewAdaptor.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import * as PMML from "./generated/www.dmg.org/PMML-4_4";
+import { Line, LinearRegressionView, Range } from './LinearRegressionView';
+
+type Props = {
+  dictionary: PMML.DataDictionaryType
+  model: PMML.RegressionModelType[]
+}
+
+const LinearRegressionViewAdaptor = (props: Props) => {
+
+  const model: PMML.RegressionModelType | undefined = getLinearRegressionModel(props);
```

Comment:
I would not pass down the entire `props` object to a function if it is not needed, I think you could extract properties from `props` and then pass just what is consumed, `model` in this case. The code is also easier to understand this way. Do you agree?
The fact that `model` is declared here but it's different from the `props.model` is a bit confusing to me at first.

Reply:
I agree and have changed accordingly. 

I've also renamed the property to `models` too as it is an array.